### PR TITLE
Correction of time and date position

### PR DESCRIPTION
**Pull request by swarakaka**

### CODE REVIEW

**Comment by kylekatarnls** [comment]
Thanks 🙏  https://github.com/briannesbitt/Carbon/blob/master/tests/Localization/CkbTest.php will need to be updated accordingly. I may try to do that this week-end.

File: tests/Localization/CkbTest.php [hunk 1]
```diff
@@ -124,108 +124,108 @@ class CkbTest extends LocalizationTestCase
         // Carbon::parse('2018-01-01 00:00:00')->ordinal('hour')
         '0',
         // Carbon::now()->subSeconds(1)->diffForHumans()
-        'پێشتر چرکە',
+        'پێش چرکەیەک',
```
Comment (kylekatarnls) [inline tests/Localization/CkbTest.php:127]:
```suggestion
        'پێشتر چرکەیەک',
```

Comment (swarakaka) [reply tests/Localization/CkbTest.php:127]: In the latest changes, this problem must have been fixed.
How can I run this test, to make sure the words are correct?

**Comment by swarakaka** [comment]
Please reopen it again, I accidentally deleted my PR.

**Comment by kylekatarnls** [comment]
I'm closing this PR in favor of #2627

We should have only 1 at a time modifying the same lines, else it will just lead to conflicts and we can't read new outputs in tests.

**Pull request closed without merging**